src/Mapping/ExtensionMetadataFactory.php

```diff
@@ -60,18 +60,18 @@ class ExtensionMetadataFactory
     protected $annotationReader;
 
     /**
-     * Initializes extension driver
-     *
-     * @param string $extensionNamespace
-     * @param object $annotationReader
+     * @var CacheItemPoolInterface|null
      */
-    public function __construct(ObjectManager $objectManager, $extensionNamespace, $annotationReader)
+    private $cachePoolItem;
```

Comment:
Just for consistency:
```suggestion
    private $cacheItemPool;
```